packages/client/src/open-feature.ts

```diff
@@ -1,8 +1,16 @@
-import { ApiEvents, EvaluationContext, FlagValue, Logger, OpenFeatureCommonAPI, ProviderEvents, ProviderMetadata, SafeLogger } from '@openfeature/shared';
-import { OpenFeatureEventEmitter } from '.';
```

Comment:
These files were both importing the barrel file, and exporting from it, which obviously is a cycle.